engine/base_client/client.py

```diff
@@ -110,12 +110,25 @@ def run_experiment(
                         continue
 
                 search_params = {**searcher.search_params}
+                ef = "default"
+                if "search_params" in search_params:
+                    ef = search_params["search_params"].get("ef", "default")
+                client_count = search_params.get("parallel", 1)
+                filter_client_count = len(parallels) > 0
+                if filter_client_count and (client_count not in parallels):
+                    print(f"\tSkipping ef runtime: {ef}; #clients {client_count}")
+                    continue
+                print(f"\tRunning ef runtime: {ef}; #clients {client_count}")
```

Comment:
Could you please explain what are you trying to do here?

Reply:
this is to allow running specific client configs. The rest is just extra logging.